ddp-workspace/projects/ddp-prion/src/app/app.module.ts

```diff
@@ -1,27 +1,43 @@
-import { NgModule, Injector, APP_INITIALIZER } from '@angular/core';
+//Angular imports
+import { APP_INITIALIZER, Injector, NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-import { LOCATION_INITIALIZED } from '@angular/common';
-import { AppRoutingModule } from './app-routing.module';
+import { CommonModule, LOCATION_INITIALIZED } from '@angular/common';
+import { RouterModule } from '@angular/router';
+import { FlexLayoutModule, FlexModule } from '@angular/flex-layout';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatCardModule } from '@angular/material/card';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIconModule } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatProgressBarModule } from '@angular/material/progress-bar';
+import { MatTableModule } from "@angular/material";
 
+//External library imports
```

Comment:
We have a ts-lint rule that demands a space after single line comments, so please change to:
```suggestion
// External library imports
```